src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -37,6 +38,13 @@ const handsontable = new Handsontable(container, {
         'remove_row',
         'undo',
         'redo',
+        {
+            key: 'paste',
+            name: 'Paste',
+            callback: function() {
+                showModalPasteBox();
```

Comment:
No need to wrap this around an anonymous function

Reply:
I think it needs to. If I do just `callback: showModalPasteBox()`, the page always show the modal box even if I didn't do Right Click > click `Paste`.

Reply:
You are supposed to pass a function, so it shouldn't have the brackets

Reply:
Done 😄